drivers/financial/app/views/financial/clients/show.haml

```diff
@@ -1,4 +1,4 @@
-- title = @client.pii_provider(user: current_user).full_name
+- title = 'Financial Clients'
```

Comment:
Maybe?
```suggestion
- title = 'Client Finances'
```

We call the records `financial_clients` but I don't think that makes sense when looking at the page.

Reply:
ah okay, I'll fix up the h3 in this page to match